native/cocos/2d/renderer/RenderEntity.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <cocos/2d/renderer/RenderDrawInfo.h>
+#include <vector>
+
+namespace cc {
+class RenderEntity
```

Comment:
Please add final if it can no be inherited.

Reply:
done